app/models/landable/access_token.rb

```diff
@@ -16,5 +17,17 @@ class AccessToken < ActiveRecord::Base
     def refresh!
       update_column :expires_at, 8.hours.from_now
     end
+
+    def can_publish?
+      permissions.include? "publish"
+    end
+
+    def can_edit?
+      permissions.include? "edit"
+    end
+
+    def can_read?
+      permissions.include? "read"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.